test_data/asm/keccakf.asm

```diff
@@ -163,6 +163,8 @@ let main: int, int[], int -> int[] = |W, input, delim| {
 };
 
 // main machine
-machine Main { 
+machine KeccakF { 
     let x;
+    let y;
+    operation keccakf<0> x -> y;
```

Comment:
Just a placeholder, but I think we need 25 columns for input and 25 columns for output right?

I mean in the syscall it's just one memory pointer for both the input and the output but the sub machine should be different right?